ledger-core-bitcoin/src/CMakeLists.txt

```diff
@@ -146,20 +87,21 @@ if(IS_IOS GREATER_EQUAL 0)
         \${BUILT_PRODUCTS_DIR}/\${PRODUCT_NAME}.framework/\${PRODUCT_NAME}
     )
 else()
-    add_library(ledger-core-bitcoin-obj OBJECT
-        ${ledger-core-bitcoin-sources}
-        ${SRC_FILES}
-        ${CORE_SRC_FILES}
-        ${HEADERS_FILES}
-        ${CORE_HEADER_FILES})
+    add_library(ledger-core-bitcoin-api-obj OBJECT ${ledger-core-bitcoin-api-sources})
+
+    add_library(ledger-core-bitcoin-obj OBJECT ${SRC_FILES} ${CORE_SRC_FILES} ${CORE_HEADER_FILES})
     target_link_libraries(ledger-core-bitcoin-obj PUBLIC ledger-core-bitcoin-interface)
 
+    # enable static analyzers if requested
+    include(StaticAnalyzers)
```

Comment:
Can you conditionally include here? The problem is that you will always include that file even if we don’t want static analyzers.

Reply:
I want to always include that file - I explained why in previous answer.

Reply:
Always including a file adds more parsing time and thus, more time to compile. Applying that rule everywhere leads to time-costly CMake configuration.

When we don’t want _any static analyzers_, we are going to pay a price for nothing here.

Reply:
Are we really talking about time consuming in `cmake` part ? Launch `cmake` command takes few seconds.

What struggle me a lot with your suggestion is :
* We lose completely the benefit of `cmake` helper; isolate `cmake` logic and reuse it as needed.
* The check we have to do to enable static analyzers is to verify if at least one of them is enabled so if we want to add a new one, we have to change the condition in all `CMakeFiles.txt`.
* We make something at first simple harder to use.